src/core/SampleBuffer.cpp

```diff
@@ -306,27 +306,13 @@ void SampleBuffer::convertIntToFloat ( int_sample_t * & _ibuf, f_cnt_t _frames,
 
 	// if reversing is on, we also reverse when
 	// scaling
-	if( m_reversed )
+	int idx = m_reversed ? ( _frames - 1 ) * _channels : 0;
```

Comment:
Just style update, can be ignored if wanted...
```suggestion
	int idx = m_reversed ? (_frames - 1) * _channels : 0;
```